src/nvim/ex_docmd.c

```diff
@@ -6788,27 +6801,51 @@ void free_cd_dir(void)
 
 #endif
 
-/*
- * Deal with the side effects of changing the current directory.
- * When "local" is TRUE then this was after an ":lcd" command.
- */
-void post_chdir(int local)
+/// Deal with the side effects of changing the current directory.
+///
+/// @param scope  Scope of the function call (editor, tab or window).
+void post_chdir(int scope)
```

Comment:
Where do you get the error? If you use something like YCM then you need to recompile and *then* you will not get the error because declarations will be regenerated, and you will only get error from YCM, but not when compiling.

In any case compiler says *where* is the non-static declaration. Remove it.